FirebaseInAppMessaging.podspec

```diff
@@ -44,6 +45,7 @@ See more product details at https://firebase.google.com/products/in-app-messagin
   s.ios.dependency 'FirebaseAnalyticsInterop', '~> 1.3'
   s.dependency 'FirebaseInstanceID', '~> 4.0'
   s.dependency 'GoogleDataTransportCCTSupport', '~> 1.0'
+	s.dependency 'FirebaseABTesting'
```

Comment:
The format seems off, I wonder if we need to specify a version and also bump up ABTesting and this podspec's version in case we forgot in next release

Reply:
Yes should depend on next ABTesting minor version and the ABTesting podspec should be bumped.